packages/manager/apps/dedicated-servers/src/components/cells/renewCell.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { DataGridTextCell } from '@ovh-ux/manager-react-components';
+import { DedicatedServer } from '@/data/types/server.type';
+import { DSBilling } from '../billingCell';
+
+type RenewCellProps = {
+  server: DedicatedServer;
+  t: (name: string) => string;
```

Comment:
can you use `useTranslation` directly inside the component or take the label as a prop ?